pkg/controller/bundle_sync_task.go

```diff
@@ -385,6 +392,9 @@ func (st *bundleSyncTask) handleProcessResult(retriable bool, processErr error)
 	}
 
 	if statusUpdated || finalizersUpdated {
+		bytes, _ := json.Marshal(st.bundle)
```

Comment:
left over from testing? This would be a little spammy for info (does zap have debug level?)

Reply:
Yep, sorry leftover from testing.